app/scripts/controllers/network/network-controller.test.js

```diff
@@ -4744,6 +5115,114 @@ describe('NetworkController', () => {
       });
     });
   });
+  describe('upsertNetworkConfiguration', () => {
+    it('should add the given network configuration if its rpcURL does not match an existing configuration', async () => {
+      v4.mockImplementationOnce(() => 'network-configuration-id-1');
+      await withController(
+        {
+          state: {
+            networkConfigurations: {},
+          },
+        },
+        async ({ controller }) => {
+          const rpcUrlNetwork = {
+            chainId: undefined,
+            chainName: 'RPC',
+            rpcPrefs: undefined,
+            rpcUrl: 'rpc_url',
+            ticker: 'RPC',
+            networkConfigurationId: 'network-configuration-id-1',
```

Comment:
Probably best to omit this property here, since `upsertNetworkConfiguration` isn't expecting it.

Reply:
removed here: https://github.com/MetaMask/metamask-extension/pull/17421/commits/de1a4049a8b7966a717fce7c1dd81d3df7e55f9f#diff-d4e78728faadb83c4ed8c0d5b5b580b026993a8988156b27bf7f41c4b5217af4L5134